src/main/resources/assets/ingameinfo/lang/ru_RU.lang

```diff
@@ -1,41 +1,305 @@
+ingameinfoxml.tag.abrfinedirection.desc=Сокращенное уточненное направление игрока, на которое он смотрит (Ю, ЮЗ, З, СЗ, ...).
+ingameinfoxml.tag.directionhud.desc=Небольшой текстовый HUD направлений.
 
-# gui - tag list - player potion
+# ГУИ - список тегов - зелья игрока
+ingameinfoxml.tag.potioneffect.desc=Название эффекта зелья.
+ingameinfoxml.tag.potionduration.desc=Продолжительность эффекта зелья в секундах.
+ingameinfoxml.tag.potiondurationticks.desc=Продолжительность эффекта в тактах.
+ingameinfoxml.tag.potionicon.desc=Значок эффекта зелья.
+ingameinfoxml.tag.potionlargeicon.desc=Большой значок эффекта зелья.
 
-# gui - tag list - riding
+# ГУИ - список тегов - верховая езда
+ingameinfoxml.tag.ridinghorse.desc=§eИстина§r если игрок верхом на лошади, §eложно§r в противном случае.
+ingameinfoxml.tag.horsehealth.desc=Здоровье лошади.
+ingameinfoxml.tag.horsemaxhealth.desc=Максимальное здоровье лошади.
+ingameinfoxml.tag.horsespeed.desc=Скорость лошади.
+ingameinfoxml.tag.horsejumpstrength.desc=Сила прыжка лошади.
 
-# gui - tag list - time
+# ГУИ - список тегов - время
+ingameinfoxml.tag.day.desc=Minecraft день, начинается от 0.
+ingameinfoxml.tag.mctimeh.desc=Minecraft час.
+ingameinfoxml.tag.mctimem.desc=Minecraft минута.
+ingameinfoxml.tag.mctime24.desc=Minecraft время (24 часовой формат).
+ingameinfoxml.tag.mctime12.desc=Minecraft время (12 часовой формат).
+ingameinfoxml.tag.rltime24.desc=Реальное время (24 часовой формат).
+ingameinfoxml.tag.rltime12.desc=Реальное время (12 часовой формат).
 
-# gui - tag list - world
+# ГУИ - список тегов - мир
+ingameinfoxml.tag.worldname.desc=Имя мира.
+ingameinfoxml.tag.worldsize.desc=Размер мира.
+ingameinfoxml.tag.worldsizemb.desc=Размер мира в MiB.
+ingameinfoxml.tag.seed.desc=Зерно мера.
+ingameinfoxml.tag.difficulty.desc=Игровая сложность мира.
+ingameinfoxml.tag.difficultyid.desc=ID игровой сложности мира.
+ingameinfoxml.tag.dimension.desc=Название измерения.
+ingameinfoxml.tag.dimensionid.desc=ID измерения.
+ingameinfoxml.tag.biome.desc=Имя биома.
+ingameinfoxml.tag.biomeid.desc=ID биома.
+ingameinfoxml.tag.daytime.desc=§eИстина§r если дневное время, §eложно§r в противном случае.
+ingameinfoxml.tag.raining.desc=§eИстина§r если идет дождь, §eложно§r в противном случае.
+ingameinfoxml.tag.thundering.desc=§eИстина§r если идет гроза, §eложно§r в противном случае.
+ingameinfoxml.tag.snowing.desc=§eИстина§r если идет снег, §eложно§r в противном случае.
+ingameinfoxml.tag.nextrain.desc=Время до следующего дождя.
+ingameinfoxml.tag.slimes.desc=§eИстина§r если текущий чанк является слизневым, §eложно§r в противном случае.
+ingameinfoxml.tag.hardcore.desc=§eИстина§r если мир находится в hardcore режиме, §eложно§r в противном случае.
+ingameinfoxml.tag.temperature.desc=Температура биома.
+ingameinfoxml.tag.localtemperature.desc=Температура биома в вашем положении.
+ingameinfoxml.tag.humidity.desc=Влажность биома.
 
-# keys
+# ГУИ - список тегов - интеграция - blood magic
```

Comment:
The plugins are no longer part of IGI and will be released as a separate addon/plugin eventually. You can use [this](http://mc.lunatri.us/translate?src=https://raw.githubusercontent.com/Lunatrius/InGame-Info-XML/master/src/main/resources/assets/ingameinfo/lang/en_US.lang&dst=https://raw.githubusercontent.com/MJaroslav/InGame-Info-XML/patch-1/src/main/resources/assets/ingameinfo/lang/ru_RU.lang) tool to make sure you're translating the latest version of the file.